src/docs/patterns/appbar.mdx

```diff
@@ -1,45 +1,133 @@
+        }
+        react={
+            'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/collapsible-app-bar/index.tsx&initialpath=collapsible'
+        }
+        ionic
+        reactNative
+    />
+    <Divider />
+</React.Fragment>
+
+# Search Bar
 
 <ImageGrid
-    images={[Image3, Image4]}
+    regularWidth
+    images={[Search]}
     caption={'When the search icon is clicked, a white search bar will slide in and cover the default tool bar. '}
 />
-<DemoCard repository={'search-bar'} angular={'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fsearch-bar%2Fsearch-bar.component.ts'} react={'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'} ionic reactNative />
 
-A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white Search Bar slides in from the right to cover the default toolbar. As the user enters text into the search field, the data is filtered in real-time.
-
-<Divider />
+A very common pattern seen in many types of applications is the ability to search for data on a page. This is achieved by placing a search icon/button in the app bar. When this button is clicked, a white search bar slides in from the right to cover the default app bar. As the user enters text into the search field, the data is filtered in real-time. A back icon will close the Search Bar and clear the search results.
+
+If your search is not a global-level search, you can place a smaller search input near the content being searched (e.g., a card header).
+
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[MiniSearch]}
+        caption={'Use a contextual search input if your search functionality is not at the page level.'}
+    />
+    <DemoCard
+        repository={'search-bar'}
+        angular={
+            'https://stackblitz.com/github/pxblue/angular-design-patterns?file=src%2Fapp%2Fpages%2Fapp-bar%2Fsearch-bar%2Fsearch-bar.component.ts'
+        }
+        react={
+            'https://codesandbox.io/s/github/pxblue/react-design-patterns/tree/master?file=/src/pages/app-bar/search-bar/index.tsx&initialpath=search'
+        }
+        ionic
+        reactNative
+    />
+</React.Fragment>
+
+## Auto-complete
+
+For more advanced searching, you may wish to include an auto-complete feature. You can implement autocomplete by including a contextual menu below the search bar with suggested search items. Alternatively, you can use type-ahead to automatically update the recommended search query in the search field as the user types (Google search does this).
+
+<React.Fragment>
+    <ImageGrid
+        regularWidth
+        images={[AutoVariants]}
```

Comment:
Don't like the elevation; see figma for a potential fix (or you can do what docit is doing)